### ISSUE TITLE
Intersect two hash tables

### ISSUE THREAD

**Issue by glogsdon1** [gmarcais/Jellyfish#144]
Hi,

I generated k-mer hash tables from two different datasets, and I would like to intersect them to find k-mers that are present in both. Is it possible to do this (perhaps using the query subcommand)?

Thanks,
Glennis

**Comment by gmarcais** [comment]
Yes. Check the merge command in the latest 2.3.0 version. The min switch should basically do what you want.
